src/Sql/Sql/ChangeLog.md

```diff
@@ -27,6 +27,7 @@
 ## Version 5.1.0
 * Added cross-subscription support for `Copy-AzSqlInstanceDatabase`, `Move-AzSqlInstanceDatabase`
 * Added new parameter SecondaryType to `Add-AzSqlDatabaseFromFailoverGroup`
+* Added new parameter AuthenticationMetadata to `New-AzSqlInstance` and `Set-AzSqlInstance`
```

Comment:
Please move the change log to top of **Upcoming Release**

Reply:
Hmm it seems Git merged it badly after pulling in the latest changes from main branch...
I've moved it to Upcoming release section